src/main/java/Duke.java

```diff
@@ -1,10 +1,161 @@
+import java.util.Scanner;
 public class Duke {
+    public static int MAX_TASK = 100;
+    public static String logo = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    public static String PROMETHEES =    " ____  ____  ____  _      _____ _____  _     _____  _____  ____\n"
+                                        +"/  __\\/  __\\/  _ \\/ \\__/|/  __//__ __\\/ \\ /|/  __/ /  __/ / ___\\\n"
+                                        +"|  \\/||  \\/|| / \\|| |\\/|||  \\    / \\  | |_|||  \\   |  \\   |    \\\n"
+                                        +"|  __/|    /| \\_/|| |  |||  /_   | |  | | |||  /_  |  /_  \\___ |\n"
+                                        +"\\_/   \\_/ \\_\\____/\\_/  \\|\\____\\  \\_/  \\_/ \\|\\____\\ \\____\\ \\____/\n";
+    public static String HORIZONTAL = "--------------------------------------";
+    public static String HELLO_GREET =
+            " My Son, I am Promethees, who had devoted his liver to liberate humanity from the Olympian's oppression\n"
+                    +" Tell me, what can I do for you?";
+    public static String GOOD_BYE = " Mission accomplished!";
+    public static String INSTRUCTION = "Invalid Command! Available Commands: bye, list, todo, deadline, event, done.";
+    public static int TODO_INDEX = 5;
+    public static int DEADLINE_INDEX = 9;
+    public static int EVENT_INDEX = 6;
```

Comment:
Since all these values are immutable and basically constant in the program- they can be declared as 'final.'